purchase_vendor_bill_breakdown/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,6 @@
+This module offers a solution for the following scenario:
+
+* We purchase a **Toolbox** product from a supplier
+* Supplier delivers us the Toolbox but issues an invoice with **Box**, **Wrench**, **Screwdriver** and **Hummer** products in line
```

Comment:
* Supplier delivers us the Toolbox but issues an invoice with **1x Box**, **2x Wrench**, **5x Screwdriver** and **1x Hammer** products in line